### DISCUSSION TITLE
Is this snap current?

### DISCUSSION THREAD

**Discussion by Minnietheandroid** [searx/searx#2851] [General]
Hi.
Congratulations on the 1.0
Very enthusiastic about searx right now, and trying out the install options. 
I was just wondering, is the snap at https://snapcraft.io/searx related to the source here?
Does anyone know if it's still maintained? Any info would be great.
It isn't working which is a shame, searx seems like a great snap or flatpak candidate.

Thank you :)

**Comment by beedell-roke** [comment]
I am quite confident that the intent of this project is provision of a search-engine, not development of a web-browser, but are you certain that that browser fulfills what you require anyway?
